kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -182,12 +186,13 @@ define([
             this.ERROR_COMM_CHANNEL_NOT_INIT = 'Comm channel not initialized, not sending message.';
 
             // set up validators for incoming backend job messages
-            this.validationFn = {};
-            this.validationFn[BACKEND_RESPONSES.INFO] = Jobs.isValidJobInfoObject;
-            this.validationFn[BACKEND_RESPONSES.LOGS] = Jobs.isValidJobLogsObject;
-            this.validationFn[BACKEND_RESPONSES.STATUS] = Jobs.isValidBackendJobStateObject;
-            this.validationFn[BACKEND_RESPONSES.STATUS_ALL] = Jobs.isValidBackendJobStateObject;
-            this.validationFn[BACKEND_RESPONSES.RETRY] = Jobs.isValidJobRetryObject;
+            this.validationFn = {
+                [BACKEND_RESPONSES.INFO]: Jobs.isValidJobInfoObject,
```

Comment:
pretty excited to find that it's possible to interpolate object keys using this shortcut!

Reply:
I'm curious to see if that'll build. Charles found a case where doing more recent ES2018-isms broke the build tool. I wonder if we can use Webpack or something instead of `r.js`.

Reply:
Looks like it built! :tada:
I think I love this. Had no idea! I have some refactoring to do.